src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -1,11 +1,24 @@
 import { Breadcrumb, BreadcrumbItem, BreadcrumbLink } from '@chakra-ui/react'
+import { useMemo } from 'react'
 import type { BreadcrumbsRoute } from 'react-router-breadcrumbs-hoc'
 import withBreadcrumbs from 'react-router-breadcrumbs-hoc'
 import { Link } from 'react-router-dom'
 import { AccountLabel } from 'components/AssetHeader/AccountLabel'
+import { Text } from 'components/Text/Text'
 import { selectAssetById } from 'state/slices/selectors'
 import { useAppSelector } from 'state/store'
 
+const GetTranslatedPathPart = (props: any) => {
```

Comment:
Can we type props? Operating on the `any` here makes me uneasy.

Is it something like:

```ts
const GetTranslatedPathPart = (props: {match: ReturnType<typeof matchPath>}) => {
```

?